Networking/Extensions/StringExtension.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  StringExtension.swift
+//  Networking
+//
+//  Created by Pablo Giorgi on 5/9/17.
+//  Copyright © 2017 Wolox. All rights reserved.
+//
+
+import Foundation
+
+public extension String {
```

Comment:
Since this is public, what happens if the user has Networking and Core? Won't this collide ? What would happen in that case?
I would move it to the Demo as an extension and use it there, not in the framework, and have the user use Core.